llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -448,6 +448,20 @@ struct PointerDiffInfo {
         NeedsFreeze(NeedsFreeze) {}
 };
 
+/// A pair of pointers that could overlap across a loop iteration.
```

Comment:
If the analysis has been removed from LAA, then I suggest the struct can also be moved elsewhere. Maybe VPlan.h ?

Reply:
That sounds good to me. I originally put it here so it was next to the related `PointerDiffInfo`